doc/modules/changes/20180427_gassmoeller_2

```diff
@@ -0,0 +1,5 @@
+Changed: The default value for AdditionalSeismicOutputs was changed from -1 to
+a signaling Nan. This ensures every plugin that creates seismic outputs needs
+to use a material model that actually fills those outputs.
```

Comment:
If there's no way around changing the change in column number in the depth average file, that should be documented here too.